docs/http-api/entry-api/delete_data.mdx

```diff
@@ -0,0 +1,324 @@
+
+<SwaggerComponent
+    method="DELETE"
+    path="/api/v1/b/:bucket_name/:entry_name"
+    summary="Remove a record from an entry or the entire entry"
+    description={
+        <>
+            The method removes a record from an entry or the entire entry.
+            If the `ts` query parameter is set, the record with the timestamp is removed.
+            Otherwise, the entire entry is removed.
+
+            <br/>
+            <br/>
+            If authentication is enabled, the method needs a valid API token with read access to the entry's bucket.
+            <br/>
+            <br/>
+            <strong> Changes: </strong>
+            <ul>
+                <li>
+                    Version 1.6: Initially, the method removed only the entire entry.
+                </li>
+                <li>
+                    Since 1.12, the method can remove a record with a specific timestamp by setting the `ts` query
+                    parameter.
+                </li>
+            </ul>
+        </>
+    }
+    parameters={
+        [
+            {
+                type: "path",
+                details: {
+                    name: ":bucket_name",
+                    description: "Name of bucket",
+                    isRequired: true,
+                },
+            },
+            {
+                type: "path",
+                details: {
+                    name: ":entry_name",
+                    description: "Name of entry",
+                    isRequired: true,
+                },
+            },
+            {
+                type: "query",
+                details: {
+                    name: "ts",
+                    dataType: "Integer",
+                    description: "A UNIX timestamp in microseconds. If it is set, the record with the timestamp is removed. If it is not set, the entire entry is removed.",
+                    isRequired: false,
+                },
+            }
+        ]
+    }
+    responses={[
+        {
+            status: "200",
+            message: "OK",
+            summary: "The entry has been removed",
+        },
+        {
+            status: "401",
+            message: "Unauthorized",
+            summary: "Access token is invalid or empty",
+        },
+        {
+            status: "403",
+            message: "Forbidden",
+            summary: "Access token doesn't have the necessary permissions",
+        },
+        {
+            status: "404",
+            message: "Not Found",
+            summary: "The bucket, entry or record doesn't exist",
+        },
+    ]}
+
+/>
+
+<SwaggerComponent
+    method="DELETE"
+    path="/api/v1/b/:bucket_name/:entry_name/batch"
+    summary="Remove a batch of records from an entry"
+    description={
+        <>
+            The method removes a batch of records from an entry. The list of records to remove is sent in the headers of
+            the request.
+            See the Batch Protocol section for more details.
```

Comment:
If possible adding link to "http-api/entry-api#batch-protocol"

Reply:
I didn't find how =( This must be a versioned link and it works only for md links like `[...](..)`.  I can add the link in the introduction text of the section.